### ISSUE TITLE
Use readthedocs.org for docs

### ISSUE THREAD

**Issue by kesali** [fuelphp/docs#2]

**Comment by WanWizard** [comment]
Are you volunteering to convert everything?

**Comment by sagikazarmark** [comment]
We have talked about documentation a lot. Read the docs is indeed a decent way of organizing documentation, but I am not quite sure we have the manpower right now to redesign the whole documentation concept. Also, documentation is not that simple in case of FuelPHP since it has multiple components. It is not decided yet whether these components will strictly be versioned together or not. There are other questions as well related to package architecture which must be cleared as well.

So I would say RTD is currently not on our scope, but who knows...

**Comment by kesali** [comment]
Yes I know team busy with other things. This is more like let you guys know there is an option.
